requirements.txt

```diff
@@ -1,14 +1,17 @@
 openslide-python==1.1.2
 scikit-image~=0.19.2
-scikit-learn==0.24.1
-numpy~=1.20.3
+scikit-learn~=1.2.1
+numpy~=1.23.5
 scipy==1.10.0
-matplotlib==3.3.4
+matplotlib~=3.7.1
 dill==0.3.3
 pytest~=7.1.3
 importlib-resources
 typing-extensions
-
-requests~=2.22.0
-Pillow~=8.4.0
-setuptools~=49.2.1
\ No newline at end of file
+lazy-property
+requests~=2.28.1
+Pillow~=9.4.0
+setuptools~=65.6.3
+shapely~=2.0.1
+zlib~=1.2.13
+openslide~=20171122
```

Comment:
Is this only for Windows build?

Reply:
nah it's probably the autofill of requirments.txt via IDE when I code in Windows. The shapely works for linux and windows.